src/pages/home.html

```diff
@@ -0,0 +1,1926 @@
+                            <span>هدیه ای ارزشمند</span>
+                        </h3>
+                        <p
+                            class="flex flex-col whitespace-nowrap text-base leading-8 text-bali-hai"
+                        >
+                            <span>با چاپ عکس های خود</span>
+                            <span>همیشه خاطرات روز های خوب و زیبایتان را</span>
+                            <span>زنده و ماندگار نگه دارید</span>
+                        </p>
+
+                        <a
+                            href="#?"
+                            class="relative z-10 flex items-center justify-center gap-3 bg-bondi-blue px-5 py-5.5 text-link-water"
+                        >
+                            <img
+                                src="/src/assets/images/home/bg-design4.png"
+                                alt=""
+                                class="absolute top-0 z-20 h-full w-full bg-cover bg-center"
+                            />
+                            <p class="text-lg font-bold">چاپ آنلاین عکس</p>
+                            <svg
+                                width="24"
+                                height="24"
+                                viewBox="0 0 24 24"
+                                fill="none"
+                                xmlns="http://www.w3.org/2000/svg"
+                            >
+                                <circle
+                                    cx="12"
+                                    cy="12"
+                                    r="9"
+                                    stroke="#ffffff"
+                                    stroke-width="1.5"
+                                    stroke-linecap="round"
+                                    stroke-linejoin="round"
+                                />
+                                <path
+                                    d="M12 9L9 12M9 12L12 15M9 12H15"
+                                    stroke="#ffffff"
+                                    stroke-width="1.5"
+                                    stroke-linecap="round"
+                                    stroke-linejoin="round"
+                                />
+                            </svg>
+                        </a>
+                    </div>
+                    <img
+                        src="/src/assets/images/home/asterisk.png"
+                        alt="عکس *"
+                        class="absolute bottom-0 left-0 h-[88px] w-[91px]"
+                    />
+                </div>
+            </section>
+            <!-- gallery section -->
+
+            <!-- services section -->
+            <section class="mx-auto mb-20 max-w-7xl">
+                <div class="relative flex items-center justify-center">
+                    <img
+                        src="/src/assets/images/home/oranges.png"
+                        alt=""
+                        class="absolute -top-[30%] right-[39%] h-[35px] w-[35px]"
```

Comment:
`-top-30% right-39%` is not trustable on responsive. Limit the parent size and boundary then use trustable values.

For example:
-div: `flex justify-center`
--- div: `relative py-8`
------ h3
------ img: `absolute top-0 right-0 translate-x-full`